cv/qc-cv.obo

```diff
@@ -539,14 +564,1316 @@ is_a: QC:4000009 ! ID based
+is_a: QC:4000003 ! single value
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+synonym: "MS2-4B" NARROW []
+
+[Term]
+id: QC:4000123
+name: Explained precursor intensity third quarter
+def: "Fraction of identified MS2 in the third quarter of MS2 sorted by precursor intensity." [PSI:QC]
+comment: Higher fractions of identified MS2 spectra indicate the efficiency of detection and sampling
+is_a: QC:4000003 ! single value
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+synonym: "MS2-4C" NARROW []
+
+[Term]
+id: QC:4000124
+name: Explained precursor intensity fourth quarter
+def: "Fraction of identified MS2 in the fourth quarter of MS2 sorted by precursor intensity." [PSI:QC]
+comment: Higher fractions of identified MS2 spectra indicate the efficiency of detection and sampling
+is_a: QC:4000003 ! single value
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+synonym: "MS2-4D" NARROW []
+
+[Term]
+id: QC:4000125
+name: Extent of identified precursor intensity
+def: "Ratio of 95th over 5th percentile of precursor intensity for identified peptides" [PSI:QC]
+comment: Can be used to approximate the dynamic range of signal
+is_a: QC:4000003 ! single value
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+property_value: has_units STATO:0000300
+synonym: "MS1-3A" NARROW []
+
+[Term]
+id: QC:4000126
+name: Precursor intensity range of identified MS2
+def: "Minimum and maximum precursor intensity recorded and identified." [PSI:QC]
+comment: The intensity range of the identified precursors informs about the dynamic range of the acquisition. 
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+is_a: QC:4000004 ! n-tuple
+property_value: has_units MS:1003085
+
+[Term]
+id: QC:4000127
+name: Precursor intensity of identified MS2 Q1, Q2, Q3
+def: "From the distribution of precursor intensity of identified MS2, the quartiles Q1, Q2, Q3" [PSI:QC] 
+comment: The (un)identified precursor intensity distribution can aid the interpretation of overall identification success.
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+is_a: QC:4000004 ! n-tuple
+property_value: has_units STATO:0000167
+property_value: has_units STATO:0000574
+property_value: has_units STATO:0000170
+
+[Term]
+id: QC:4000128
+name: Precursor intensity of unidentified MS2 Q1, Q2, Q3
+def: "From the distribution of precursor intensity of unidentified MS2, the quartiles Q1, Q2, Q3" [PSI:QC] 
+comment: The (un)identified precursor intensity distribution can aid the interpretation of overall identification success.
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+is_a: QC:4000004 ! n-tuple
+property_value: has_units STATO:0000167
+property_value: has_units STATO:0000574
+property_value: has_units STATO:0000170
+
+[Term]
+id: QC:4000129
+name: Median S/N for MS1 spectra in RT range in which the middle half of the peptides are identified
```

Comment:
puhh.. thats a mouthful.

and is S/N based on intensity?
`middle half` is also weird...

`Median S/N of MS1 spectra whose RT is between Q1-Q3 of identified MS2`?